examples/pipelines/commoncrawl/components/filter_webpage_urls/fondant_component.yaml

```diff
@@ -0,0 +1,25 @@
+name: Filter webpage urls
+description: Component that filters webpage urls based on given blacklist categories
+# image: ghcr.io/ml6team/filter_webpage_urls:latest
+image: 050243278346.dkr.ecr.us-east-1.amazonaws.com/filter_webpage_urls:latest
+
+consumes:
+  webpage:
+    fields:
+      url:
+        type: string
+      html:
+        type: string
```

Comment:
Do you actually need the html? I don't think you're using it.

Reply:
no, we don't use it but we need to keep the whole dataset for the next component in the pipeline.